docs/swagger/spec.yml

```diff
@@ -838,15 +923,60 @@ definitions:
       - $ref: '#/definitions/BaseModel'
     required:
       - name
-      - slugLabel
+      - id
     properties:
       name:
         type: string
         description: Display name for organization
-      slugLabel:
+      id:
+        type: string
+        description: UUID for the organization
+        readOnly: true
+  UserRole:
+    type: object
+    properties:
+      id:
+        type: string
+        description: User id
+        readOnly: true
+      role:
+        type: string
+        description: User role in organization
+      createdAt:
+        type: string
+        description: timestamp of object creation
+        format: date-time
+        readOnly: true
+      modifiedAt:
         type: string
-        description: URL safe version of name
+        description: timestamp of object modificiation
+        format: date-time
         readOnly: true
+    required:
+      - id
+      - role
+      - createdAt
+      - modifiedAt
+  UserRolePaginated:
+    allOf:
+      - $ref: '#/definitions/PaginatedResponse'
+    properties:
+      results:
+        type: array
+        items:
+          $ref: '#/definitions/UserRole'
+  UserRoleCreate:
```

Comment:
I don't think this needs to be a separate model here, just don't set the id and role as readonly